10x/count/count.wdl

```diff
@@ -103,12 +106,7 @@ task chunk_reads_main {
 
   command <<<
     
-    # Move the reads files to a path in the cwd. Note this needs to be a hard link
-    # Martian will deference soft links.
-    mkdir read_path
-    for read_file in '${read_path}'/*; do
-      ln "$read_file" read_path/"$(basename $read_file)"
-    done
+    mkdir read_path && tar -xf ${read_path} -C read_path/
```

Comment:
Hmm, File read_path is described above as a directory containing fastqs. As suggested in comment on setup_chunks, could we use an array of fastqs instead of a tarball here?

Reply:
For now, I adjusted the comment to point out that there is a tarball here.